tcl.html.markdown

```diff
@@ -0,0 +1,446 @@
+---
+language: Tcl
+contributors:

```

Comment:
Above this line you should have ```filename: learntcl.tcl``` (I'm not sure what the file extension for a TCL file is, I am just guessing .tcl so change that if not correct.)